src/software/geom/voronoi_util.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <boost/polygon/voronoi.hpp>
+
+#include "software/geom/circle.h"
+#include "software/geom/point.h"
+#include "software/geom/rectangle.h"
+
+using boost::polygon::voronoi_builder;
+using boost::polygon::voronoi_diagram;
+using boost::polygon::voronoi_vertex;
+
+struct VoronoiDiagram
```

Comment:
seems like this should just a class at this point?

Reply:
yeah fair enough. Will change. I'm just going to put the other util functions in this file as class methods too unless there are any objections?

Reply:
yeah makes sense, let's rename the file as well to match the class name.

Reply:
Done